src/openforms/formio/typing/custom.py

```diff
@@ -7,3 +7,7 @@
 class DateComponent(Component):
     datePicker: NotRequired[DatePickerConfig]
     customOptions: NotRequired[DatePickerCustomOptions]
+
+
+class AddressNLComponent(Component):
+    deriveAddress: NotRequired[bool]
```

Comment:
so with the data migration + import converter, we can now effectively enforce that this property is present:

```suggestion
class AddressNLComponent(Component):
    deriveAddress: bool
```